RadixWallet/Clients/TokenPriceClient/TokenPriceClient+Live.swift

```diff
@@ -4,11 +4,8 @@ extension TokenPricesClient {
 		@Dependency(\.jsonDecoder) var jsonDecoder
 		@Dependency(\.jsonEncoder) var jsonEncoder
 		@Dependency(\.cacheClient) var cacheClient
-		#if DEBUG
```

Comment:
Opt to always use prod token price service to facilitate testing. Dev endpoint might not have all prices available or latest ones.